plugins/base/src/main/kotlin/transformers/documentables/SuppressedDocumentableFilterTransformer.kt

```diff
@@ -0,0 +1,51 @@
+package org.jetbrains.dokka.base.transformers.documentables
+
+import org.jetbrains.dokka.DokkaConfiguration
+import org.jetbrains.dokka.DokkaConfiguration.DokkaSourceSet
+import org.jetbrains.dokka.model.*
+import org.jetbrains.dokka.plugability.DokkaContext
+import org.jetbrains.dokka.transformers.documentation.PreMergeDocumentableTransformer
+import java.io.File
+
+class SuppressedDocumentableFilterTransformer(val context: DokkaContext) : PreMergeDocumentableTransformer {
+    override fun invoke(modules: List<DModule>): List<DModule> {
+        return modules.mapNotNull { module -> filterModule(module) }
+    }
```

Comment:
This looks like a common pattern. Do you think it is worth extracting it to some base class extending `PreMergeDocumentableTransformer` that abstracts recursive invocation of predicates for different Documentables?

Reply:
With the given requirements, I would say that it's OK since there are not soo lines of code involved. But we should definitely keep this idea in our head if we have to implement more "FilterTransformer" classes 👍